sqlx-core/src/migrate/migration_type.rs

```diff
@@ -13,6 +13,18 @@ pub enum MigrationType {
     ReversibleDown,
 }
 
+// It seems that deriving PartialEq leads to some weird lifetime bug I don't know why:
+//
+//   `sqlx-core/src/postgres/copy.rs` then fails to compile with:
+//   `associated function was supposed to return data with lifetime `'c` but it is returning data with lifetime `'1``
```

Comment:
This is actually an incremental compilation bug that was just recently fixed in 1.62.1: https://github.com/rust-lang/rust/issues/98890

It was breaking our CI for a while in random spots too.

Reply:
done